pkgs/development/libraries/gstreamer/legacy/gstreamer/default.nix

```diff
@@ -16,6 +16,7 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ autoreconfHook flex perl pkgconfig libintl bison2 glib ];
+  buildInputs = stdenv.lib.optional stdenv.isDarwin libunwind;
```

Comment:
Is this intentional, or just a result of confusion of `gstreamer` with `gst_all_1.gstreamer`?